release-notes.1.2.0.md

```diff
@@ -0,0 +1,69 @@
+# vspd 1.2.0
+
+vspd v1.2.0 contains all development work completed since v1.1.0 (March 2022).
+Since then, 80 commits have been produced and merged by 3 contributors.
+All commits included in this release can be viewed on GitHub
+[here](https://github.com/decred/vspd/compare/release-v1.1.0...release-v1.2.0).
+
+## Dependencies
+
+vspd 1.2.0 must be built with go 1.19 or later, and requires:
+
+- dcrd 1.8.0
+- dcrwallet 1.8.0
+
+When deploying vspd to production, always use release versions of all binaries.
+Neither vspd nor its dependencies should be built from master when handling
+mainnet tickets.
+
+## Recommended Upgrade Path
+
+The upgrade path below includes vspd downtime, during which clients will not be
+able to register new tickets, check their ticket status, or update their voting
+preferences. Voting on tickets already registered with the VSP will not be
+interrupted. You may wish to put up a temporary maintenance webpage or announce
+downtime in public channels.
+
+1. Build vspd from the 1.2.0 release tag. Build dcrwallet and dcrd from their
+   1.8.0 release tags.
+1. Stop vspd.
+1. Stop the instance of dcrd running on the vspd server.
+1. **Make a backup of the vspd database file in case rollback is required.**
+1. Install new dcrd binary on the vspd server, start it and it will begin
```

Comment:
This is just a minor nit, but it seems a little big different as written when compared to rest because they are direct imperative statements whereas this is kind of a mix.  Perhaps the following?

```suggestion
1. Install new dcrd binary on the vspd server and start it to begin
```

Reply:
Nice suggestion, I'll change that